src/ess/amor/data.py

```diff
@@ -19,6 +19,13 @@ def _make_pooch():
         registry={
             "reference.nxs": "md5:56d493c8051e1c5c86fb7a95f8ec643b",
             "sample.nxs": "md5:4e07ccc87b5c6549e190bc372c298e83",
+            "amor2023n000608.hdf": "md5:e3a8b5b8495bb9ab2173848096df49d6",
+            "amor2023n000609.hdf": "md5:d899d65f684cade2905f203f7d0fb326",
+            "amor2023n000610.hdf": "md5:c9367d49079edcd17fa0b98e33326b05",
+            "amor2023n000611.hdf": "md5:9da41177269faac0d936806393427837",
+            "amor2023n000612.hdf": "md5:602f1bfcdbc1f618133c93a117d05f12",
+            "amor2023n000613.hdf": "md5:ba0fbcbf0b45756a269eb3e943371ced",
+            "amor2023n000614.hdf": "md5:18e8a755d6fd671758fe726de058e707",
```

Comment:
If you have any info about these files, please add them in a comment. What is each file? Where did we get them from, etc.?

Reply:
Info missing.

Reply:
I have asked Jochen about that but got no answer yet.
I'll add it when I get it. But I don't think that should block the PR.